### PR TITLE
Updated the "getting started" doc section

### PR DESCRIPTION
**Pull request by eugmandel**
What is in the PR
=============

1. New "Typical Workflow" article
2. Removed most of the content from the page that describes the init command, since our requirement is that this flow must require no tutorial.

How to review the Typical Workflow article
===============================

Imagine that you are a new user who just completed running `great_expectations init` for the first time. You have seen the initial promise of GE on your data, but you don't have a good idea of what a typical day with GE will look like and how to get there.

This article is for you.

Please review for clarity, helpfulness, style, grammar.

### CODE REVIEW

File: docs/getting_started/typical_workflow.rst [hunk 1]
```diff
@@ -3,19 +3,25 @@
...
 The article focuses on the "What" and the "Why" of each step in this workflow, and touches on the "How" only briefly. The exact details of configuring and executing these steps are intentionally left out - they can be found in the tutorials and reference linked from each section.
 
+If you have not installed Great Expectations and executed the CLI init command, as described in this :ref:`tutorial<tutorial_init>`, we recommend you do so before reading the rest of the article. This will make a lot of concepts mentioned below more familiar to you.
+
 
 Setting up a project
 ----------------------------------------
 
-To use Great Expectations in a new data project, a Data Context must be initialized. You will see references to Data Context throughout the documentation. Data Context simply represents a Great Expectations project.
+To use Great Expectations in a new data project, a :ref:`Data Context<data_context>` must be initialized. You will see references to Data Context throughout the documentation. Data Context simply represents a Great Expectations project.
```
Comment (jcampbell) [inline docs/getting_started/typical_workflow.rst]:
```suggestion
To use Great Expectations in a new data project, a :ref:`Data Context<data_context>` should be initialized. You will see references to the Data Context throughout the documentation. A Data Context provides the core services used in a Great Expectations project.
```

File: docs/getting_started/typical_workflow.rst [hunk 2]
```diff
@@ -74,7 +80,7 @@ When Great Expectations evaluates this expectation against a dataset that has a
 
 This operation can be executed in one of the following compute environments (engines): Pandas, PySpark and an SQL database. This means that the evaluated dataset (or batch of data) can be a Pandas DataFrame, a PySpark DataFrame or a query result set.
```
Comment (jcampbell) [inline docs/getting_started/typical_workflow.rst]:
```suggestion
```

File: docs/getting_started/typical_workflow.rst [hunk 3]
```diff
@@ -74,7 +80,7 @@ When Great Expectations evaluates this expectation against a dataset that has a
...
 
-A Datasource is an object Great Expectations uses to connect to a compute environment (e.g., a Postgres database on a particular host).
+A :ref:`Data Datasource<datasource>` is an object Great Expectations uses to connect to a compute environment (e.g., a Postgres database on a particular host).
```
Comment (jcampbell) [inline docs/getting_started/typical_workflow.rst]: Using glossary language

```suggestion
A :ref:`Data Datasource<datasource>` a connection to a compute environment (a backend such as Pandas, Spark, or a SQL-compatible database) and one or more storage environments.
```

File: docs/getting_started/typical_workflow.rst [hunk 4]
```diff
@@ -139,9 +145,9 @@ To set up Data Docs for a project, a “data documentation site” (a static HTM
 
 Multiple sites can be configured inside a project, each suitable for a particular data documentation use case. For example, some data teams use one site that has expectations and validation results from all the runs of their data pipeline for monitoring the pipeline's health, and another site that has only the expectations for communicating with their client (similar to API documentation in software development).
 
-By default Data Docs sites' files are published to the local filesystem in `great_expectations/uncommitted/data_docs/` directory. To make the site available to the team, a team member can be configure it to publish to a shared location, such as a S3 or GCS.
+By default Data Docs sites' files are published to the local filesystem in `great_expectations/uncommitted/data_docs/` directory. To make the site available to the team, a team member can be configure it to publish to a shared location, such as a :ref:`S3<publishing_data_docs_to_s3>` or GCS.
```
Comment (jcampbell) [inline docs/getting_started/typical_workflow.rst]:
```suggestion
By default Data Docs sites' files are published to the local filesystem in the `great_expectations/uncommitted/data_docs/` directory. To make a site available more broadly, a team member could configure Great Expectations to publish the site to a shared location, such as a :ref:`S3<publishing_data_docs_to_s3>` or GCS.

File: docs/getting_started/typical_workflow.rst [hunk 5]
```diff
@@ -139,9 +145,9 @@ To set up Data Docs for a project, a “data documentation site” (a static HTM
...
 
-All the Data Docs sites that your project has are defined in the ``great_expectations/great_expectations.yml`` configuration file. The site's configuration defines what they should display and where they are hosted. Data Docs is very customizable, but the details are beyond this article's scope.
+All the Data Docs sites that your project has are defined in the ``great_expectations/great_expectations.yml`` configuration file. The site's configuration defines what they should display and where they are hosted. Data Docs is very customizable, but the details are beyond this article's scope. See this article for details: :ref:`Data Docs Reference<data_docs_reference>`.
```
Comment (jcampbell) [inline docs/getting_started/typical_workflow.rst]:
```suggestion
All of your project's Data Docs sites are defined in the ``great_expectations/great_expectations.yml`` configuration file. The site's configuration defines what to compile and where to store results. Data Docs is very customizable;.see the :ref:`Data Docs Reference<data_docs_reference>` for more information.
```

File: docs/getting_started/typical_workflow.rst [hunk 6]
```diff
@@ -202,29 +208,102 @@ The generated Jupyter notebook can be discarded, since it is auto-generated.
 
 
 
-Deploying validation into a pipeline
-----------------------------------------
+Deploying automated testing into a pipeline
+-------------------------------------------
+
+So far, your team and you used Great Expectations to capture and document your expectations from the data.
```
Comment (jcampbell) [inline docs/getting_started/typical_workflow.rst]: This is an abrupt change in voice. I preferred the old tone, but we should settle before making line-in line-out changes

**Review by jcampbell — DISMISSED** [review]
Some inline changes; would like to visit change in voice mid stream.

File: docs/getting_started/typical_workflow.rst (hunk 6 above)
Comment (eugmandel) [reply docs/getting_started/typical_workflow.rst]: Great - thank you - will update the second half to use the same (more 3rd person/non-prescriptive) voice as the first half.

File: docs/getting_started/typical_workflow.rst [hunk 7]
```diff
@@ -72,9 +78,8 @@ For example, imagine tht we have a movie ratings table in the database. This exp
 
 When Great Expectations evaluates this expectation against a dataset that has a column named "rating", it returns a validation result saying whether the data meets the expectation.
 
-This operation can be executed in one of the following compute environments (engines): Pandas, PySpark and an SQL database. This means that the evaluated dataset (or batch of data) can be a Pandas DataFrame, a PySpark DataFrame or a query result set.
```
Comment (Aylr) [inline docs/getting_started/typical_workflow.rst:75]: typo above: `tht` --> `that`

File: docs/getting_started/typical_workflow.rst [hunk 8]
```diff
@@ -72,9 +78,8 @@ For example, imagine tht we have a movie ratings table in the database. This exp
...
 
-A Datasource is an object Great Expectations uses to connect to a compute environment (e.g., a Postgres database on a particular host).
+A :ref:`Data Datasource<datasource>` a connection to a compute environment (a backend such as Pandas, Spark, or a SQL-compatible database) and one or more storage environments.
```
Comment (Aylr) [inline docs/getting_started/typical_workflow.rst]: "A Data Datasource a connection" --> "A Datasource is a connection"

File: docs/getting_started/typical_workflow.rst [hunk 9]
```diff
@@ -72,9 +78,8 @@ For example, imagine tht we have a movie ratings table in the database. This exp
...
 
 Each You can have multiple Datasources in a project (Data Context). This is useful if the team's pipeline consists of, for example, both a Spark cluster and a Redshift database.
```
Comment (Aylr) [inline docs/getting_started/typical_workflow.rst]:
```suggestion
You can have multiple Datasources in a project (Data Context). For example, this is useful if the team’s pipeline consists of both a Spark cluster and a Redshift database.
```

File: docs/getting_started/typical_workflow.rst [hunk 10]
```diff
@@ -72,9 +78,8 @@ For example, imagine tht we have a movie ratings table in the database. This exp
...
```
Comment (Aylr) [inline docs/getting_started/typical_workflow.rst:85]: below change:

All the Datasources that your project uses are configured in the ``great_expectations/great_expectations.yml`` configuration file of the Data Context:

to

All the Datasources that your project uses are configured in the project's configuration file (``great_expectations/great_expectations.yml``).:

**Review by Aylr — APPROVED** [review]

**Pull request merged**